CarRecognition/Source Files/Common/Services/SnackBarDisplayer.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SnackBarDisplayer.swift
+//  CarRecognition
+//
+
+import UIKit
+
+struct SnackBarDisplayer {
+    
+    private struct Constants {
+        static let attachPinErrorLabelHeight: CGFloat = 40
+        static let attachPinErrorLabelFontSize: CGFloat = 20
+        static let animationDuration = 0.3
+    }
+    
+    private static var snackBar: UILabel = {
+        let label = UILabel()
+        label.backgroundColor = .black
+        label.alpha = 0.0
+        label.layer.cornerRadius = Constants.attachPinErrorLabelHeight / 2.0
+        label.clipsToBounds = true
+        label.textColor = .white
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textAlignment = .center
+        return label
+    }()
+    
+    static func show(in view: UIView, text: String) {
+        snackBar.text = text
+        
+        view.addSubview(snackBar)
+        
+        snackBar.constraintCenterToSuperview(axis: [.horizontal], withConstant: .zero)
+        snackBar.constraintToEdges(of: view, excludingAnchors: [.top, .left, .right], withInsets: UIEdgeInsets(top: 0, left: 0, bottom: 20, right: 0))
+        NSLayoutConstraint.activate([
+            snackBar.heightAnchor.constraint(equalToConstant: Constants.attachPinErrorLabelHeight),
+            snackBar.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.9)
+        ])
+        snackBar.heightAnchor.constraint(equalToConstant: Constants.attachPinErrorLabelHeight)
+        snackBar.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.9)
+        
+        UIView.animate(withDuration: 0.3, delay: 1.0, options: [], animations: {
```

Comment:
`Constants.animationDuration` instead of `0.3`

Reply:
Sorry, I forgot about it xD